scripts/tests/test_migrate_deleted_wikis.py

```diff
@@ -0,0 +1,46 @@
+from nose.tools import *
+
+from scripts.migration.migrate_deleted_wikis import get_targets, migrate
+
+
+from tests.base import OsfTestCase
+from tests.factories import ProjectFactory, UserFactory, Auth
+
+class TestMigrateDeletedWikis(OsfTestCase):
+    def setUp(self):
+        super(TestMigrateDeletedWikis, self).setUp()
+        self.user = UserFactory()
+        self.auth = Auth(user=self.user)
+        self.project = ProjectFactory(creator=self.user)
+        # Update home wiki (which can't be deleted) an a second wiki
+        self.project.update_node_wiki('home', 'Hello world', self.auth)
+        self.project.update_node_wiki('second', 'Hola mundo', self.auth)
+        self.project.update_node_wiki('second', 'Hola mundo 2', self.auth)
+        # Delete the second wiki to populate targets
+        self.project.delete_node_wiki('second', self.auth)
```

Comment:
Since you have the new code in delete_node_wiki, why isn't this call removing the wiki page versions?